sunpy/net/jsoc/jsoc.py

```diff
@@ -461,6 +461,10 @@ def fetch(self, jsoc_response, path=None, progress=True, overwrite=False,
             A Results object
 
        """
+        for resp in jsoc_response.query_args:
+            if 'notify' not in resp:
+                raise ValueError('Email address is not specified by a.jsoc.Notify')
```

Comment:
I feel like this error message should have more details on how to fix it.

Reply:
"Pass attrs.jsoc.Notify with a JSOC registered email as an argument to fido.search" ?

Reply:
Yeah that works.